src/css/graph.css

```diff
@@ -0,0 +1,232 @@
+  justify-content: center;
+  margin-bottom: 50px;
+}
+
+.scatter-plot {
+  width: 600px;
+  height: 500px;
+  background: rgba(255, 255, 255, 0.05);
+  overflow: visible;
+}
+
+.scatter-plot-subhead {
+  position: relative;
+  margin: 30px 0;
+  text-align: center;
+}
+
+.scatter-plot-key {
+  padding: 9px 15px;
+}
+
+.scatter-plot-key span {
+  font-size: 11px;
+  line-height: 16px;
+}
+
+.scatter-plot-key span:not(:last-of-type) {
+  margin-right: 20px;
+}
+
+.key-circle::before {
+  content: '';
+  display: inline-block;
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  margin-right: 8px;
+}
+
+.key-circle.green::before {
+  background-color: #00ED89;
+}
+
+.key-circle.purple::before {
+  background-color: #C14EF8;
+}
+
+.axis-label {
+  fill: #FFF;
+  font-size: 11px;
+  line-height: 13px;
+  text-transform: uppercase;
+}
+
+.label-wrapper {
+  overflow: visible;
+}
+
+.axis-tick {
+  fill: rgba(255, 255, 255, 0.75);
+  font-size: 11px;
+  line-height: 16px;
+}
+
+.axis-line {
+  stroke: rgba(255, 255, 255, 0.1);
+  stroke-width: 0.5px;
+}
+
+.scatter-point.outlier,
+.scatter-line.outlier,
+.scatter-text.outlier {
+  display: none;
+  opacity: 50%;
+}
+
+.scatter-plot.show-outliers .scatter-line.outlier,
+.scatter-plot.show-outliers .scatter-point.outlier,
+.scatter-plot.show-outliers .scatter-text.outlier {
+  display: inline-block;
+}
+
+.scatter-line {
+  stroke: rgba(255, 255, 255, 0.2);
+  stroke-width: 0.5px;
+}
+
+.scatter-point.black {
+  fill: #00ED89;
+}
+
+.scatter-point.white {
+  fill: #C14EF8;
+}
+
+.scatter-line.hovering, .scatter-point.hovering {
+  stroke: #FFF;
+  stroke-width: 0.5px;
+}
+
+.scatter-text {
+  fill: #FFF;
+  font-size: 11px;
+  line-height: 16px;
+  cursor: default;
+}
+
+.scatter-tooltip {
+  position: absolute;
+  width: 260px;
+  margin: 20px 0 0 20px;
+  padding: 11px;
+  background-color: #FFF;
+  border: 1px solid #DCDCDC;
+  border-radius: 4px;
+  box-sizing: border-box;
+  box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.25);
+  font-size: 12px;
+  line-height: 13px;
+  color: #000;
+}
+
+.scatter-tooltip h4 {
+  margin: 0;
+  font-family: "GT America Bold";
+  font-weight: 400;
+}
+
+.scatter-tooltip table {
+  width: 100%;
+  table-layout: fixed;
+  border-collapse: collapse;
+}
+
+.scatter-tooltip tbody tr {
+  border-top: 0.5px solid #D6D6D6;
+}
+
+.scatter-tooltip th {
+  padding: 5px 0;
+  font-family: "GT America Condensed";
+  font-weight: 400;
+  text-align: right;
+  text-transform: uppercase;
+  vertical-align: bottom;
+}
+
+.scatter-tooltip td {
+  padding: 5px 0;
+  text-align: right;
+  text-transform: capitalize;
+  font-variant-numeric: tabular-nums;
+}
+
+.scatter-tooltip td:not(.scatter-tooltip-name) {
+  width: 80px;
+  font-family: "GT America Bold";
+}
+
+.scatter-tooltip .scatter-tooltip-name {
+  text-align: left;
+}
+
+/* these are custom breakpoints to make the svg work well on phones */
+@media only screen and (max-width: 670px) {
+  .scatter-plot {
+    width: 300px;
```

Comment:
Is there any reason why we aren't using 100% for width with a max-width of 600px? Wouldn't that account for these different screen sizes? 

I guess my point is that when the window width is close to 670px, a 300px wide chart looks weirdly small. Maybe it makes more sense to make the cart width responsive?

Reply:
Ah, I'll have to try that out. I suspect the problem will be resizing the viewBox, but I'll check.